docs/Developers/Newcomers_Guide.rst

```diff
@@ -87,7 +87,41 @@ Step 1. Meet the Community!
 
 To get started, the first step is to meet the community. We use gitter to
 communicate, and there the helpful community will guide you.
-Join us at `coala gitter <https://coala.io/chat>`_.
+Gitter is an instant messaging service used by developers and users of GitHub.
+Gitter uses chatrooms, where developers can join in and can talk about a
+particular topic.
+coala has 2 types of chatrooms - repository chatrooms and discussion topics.
+Repository chatrooms are related to a specific repository and
+discussion chatrooms are related to general discussion topics like
+conferences, workshops, etc.
+
+  * `coala <https://gitter.im/coala/coala>`_
+    This is the main chatroom and repository chatroom of coala/coala.
+  * `gsoc <https://gitter.im/coala/coala/gsoc>`_
+    This is where you discuss about Google Summer of Code.
+  * `coala-bears <https://gitter.im/coala/coala-bears>`_
+    Repo chatroom for coala/coala-bears.
+  * `workshop <https://gitter.im/coala/coala/workshops>`_
+    Discussions related to workshops go here.
+  * `conferences <https://gitter.im/coala/conferences>`_
+    Everything related to conferences.
+  * `perfomance <https://gitter.im/coala/performance>`_
+    Profiling and performance optimizations!
```

Comment:
The offtopic room really needs to go here, see [my original comment](https://github.com/coala/coala/pull/4219#discussion_r116158884). I would remove performance, as it's not really interesting to newcomers.